docs/docs/deploying-to-clodui.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploying to Clodui
+---
+
+[Clodui] is a static hosting service; Clodui automatically optimizes your website files for speed. This guide explains how to deploy your Gatsby website to Clodui.
+
+Below is the list of features Clodui support to improve your website speed and development efforts.
+
+- Your website served from a global CDN to reduce network latency.
+- Compress your files to Brotli or GZip and serve it based on the browser support.
+- Minifies all your HTML, CSS and JavaScript files.
+- Generates [WebP](https://developers.google.com/speed/webp) version of all your images. Images then served based on browser support.
+- Dynamic image resizes support based on `Client-Hint` headers or query string parameters.
+- Automatic form submission handling with support for webhook integration.
+- Deployments are atomic with support for instant rollback.
+
+## Steps to get started
+
+Follow these steps to deploy your website to Clodui.
+
+1. [Sign up](https://app.clodui.com/auth/signup) to create an account using your email and password.  
+   _**Note :** Don't use social login like Google or Facebook to sign up because Clodui CLI doesn't support it yet._
+2. Install the [Clodui CLI]:
+   ```bash
+   npm install @clodui/cli -g
+   ```
+3. Add environment variables `CLODUI_USERNAME`, `CLODUI_PASSWORD` and assign with Clodui username and password respectively.
```

Comment:
I assume this is being added to a `.env` file in the root of a gatsby site? I would recommend stating that. Also, where is this added? Gatsby config? a code block would help.

Reply:
I'll include the code snippet which shows this; basically, on Linux/macOS you do like this
```
export CLODUI_USERNAME=<username>
export CLODUI_PASSWORD=<password>
```